src/pages/preferences/NotificationsTab.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import TextInput from "@leafygreen-ui/text-input";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Body } from "@leafygreen-ui/typography";
+import { Radio } from "antd";
+import { Notifications } from "gql/generated/types";
+import { notificationFields } from "./fieldMaps";
+
+interface ProfileTabProps {
+  slackUsername?: string;
+  notifications?: Notifications;
+}
+export const NotificationsTab: React.FC<ProfileTabProps> = ({
+  slackUsername,
+  notifications,
+}) => {
+  const [slackUsernameField, setslackUsernameField] = useState(slackUsername);
+  const [notificationStatus, setNotificationStatus] = useState(notifications);
+
+  const hasFieldUpdates =
+    slackUsername !== slackUsernameField ||
+    notificationStatus !== notifications;
+  const omitTypename = (key, value) =>
+    key === "__typename" ? undefined : value;
+  const newPayload = JSON.parse(JSON.stringify(notifications), omitTypename);
```

Comment:
I iterate through the notification object to generate the table and __typename is included as a field so this just removes it before the fields are rendered.